src/components/ui/Text.tsx

```diff
@@ -75,10 +80,11 @@ const linkText = (
   extraSmall?: boolean,
   bold?: boolean,
   gradientFrom?: TGradientColorStops,
-  gradientTo?: TGradientColorStops
+  gradientTo?: TGradientColorStops,
+  underline?: boolean
 ) =>
   classnames(
-    textDecoration('no-underline'),
+    textDecoration(underline ? 'underline' : 'no-underline'),
```

Comment:
`no-underline` is default, no need to set it explicitly, please do:
```TS
    textDecoration({'underline': underline}),
```

Reply:
Or maybe
```TS
    textDecoration({'no-underline': !underline}),
```

Reply:
good catch, ty